docs/jwst/saturation/description.rst

```diff
@@ -34,6 +34,13 @@ The "NO_SAT_CHECK" flag is propagated to the
 PIXELDQ array in the output science data to indicate which pixels fall into
 this category.
 
+For grouped data, this step also looks for and flags data that saturated during
+the second group but did not trigger the normal saturation threshold due to the
+grouped data averaging.  This requires that the third group be saturated, and
+the first group sufficiently low that the third group would not have been expected
```

Comment:
Not sure I understand this. This only catches cases where group 3 must've been affected by a CR hit, which then caused it to be above the saturation threshold? But if it's the case that the high level in group 3 was due to a CR hit that occurred somewhere in group 3, why is group 2 then suspect? Or are you saying that the CR hit actually occurred in one of the frames of group 2, but did not drive the average for group 2 above the saturation threshold?

Reply:
The latter.  CR hit actually caused saturation in group 2, but the averaging kept it below threshold.